bin/bst.ts

```diff
@@ -27,7 +27,14 @@ program
 Global.initializeCLI().then(
     () => program.parse(process.argv)
 ).catch((error) => {
-    LoggingHelper.error(Logger, error);
+    // Request to create pipe or source failed
+    if (error.code === "ETIMEDOUT") {
+        LoggingHelper.error(Logger, "Couldn't establish connection, please try again later");
+    } else {
+        LoggingHelper.error(Logger, "Something went wrong, please try again later");
```

Comment:
Okay, one last tweak I think it should read: "Something went wrong. Please check your network connection and try again."